src/python/grpcio/grpc/_cython/_cygrpc/grpc_environment.pyx.pxi

```diff
@@ -0,0 +1,2 @@
+# Set this to True for OSS builds, False for Google3 builds
```

Comment:
Please do not reference Google3 in OSS code.

Reply:
updated doc string